docs/api/api_core.rst

```diff
@@ -0,0 +1,83 @@
+.. Low-level core functions
```

Comment:
@Dishie2498, can you please clean the comments in all of the files that contain them?